web/packages/hovercards/src/style.scss

```
@@ -4,6 +4,8 @@ $color-super-light-gray: #eee;
 $color-light-gray: #d8dbdd;
 $color-gray: #707070;
 $color-blue: #1d4fc4;
+/* stylelint-disable value-keyword-case */
```

Disable the rule for the next line only:

```suggestion
/* stylelint-disable-next-line value-keyword-case */
```